website/docs/r/cdn_endpoint_custom_domain.html.markdown

```diff
@@ -88,9 +112,10 @@ In addition to the Arguments listed above - the following Attributes are exporte
 The `timeouts` block allows you to
 specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
 
-* `create` - (Defaults to 30 minutes) Used when creating the Custom Domain for this CDN Endpoint.
-* `read` - (Defaults to 5 minutes) Used when retrieving the Custom Domain for this CDN Endpoint.
-* `delete` - (Defaults to 30 minutes) Used when deleting the Custom Domain for this CDN Endpoint.
+* `create` - (Defaults to 20 hours) Used when creating the CDN Endpoint Custom Domain.
```

Comment:
Can we change all of these back to:
```suggestion
* `create` - (Defaults to 20 hours) Used when creating the Custom Domain for this CDN Endpoint.
```

Reply:
This part is auto generated via the website-scaffolding tool.

Reply:
That doesn't mean that the phrasing is always comprehensible. This was also already [flagged in a previous PR](https://github.com/hashicorp/terraform-provider-azurerm/pull/11120#discussion_r608729614).